### PR TITLE
error handling

### PR DESCRIPTION
**Pull request by mekan-allaberdi**

### CODE REVIEW

File: src/slack/slack.js [hunk 1]
```diff
@@ -832,9 +832,9 @@ export class Slack {
   }
 
   async removeReaction(name, channel, timestamp) {
-    const {ok, error} = await this.apiCall('reactions.remove', {name, channel, timestamp}, {passError: true})
-
-    if (ok === false && error !== 'no_reaction') {
+    try {
+      await this.boltApp.client.reactions.remove({name, channel, timestamp})
```
Comment (rtrembecky) [inline src/slack/slack.js:836]: you can remove the `apiCall` function and also other functions it used, so `makeApiCall` is also not needed now

File: .babelrc [hunk 2]
```diff
@@ -4,10 +4,6 @@
     "transform-class-properties",
     "transform-es2015-modules-commonjs",
     "transform-export-extensions",
-    "transform-object-rest-spread",
-    ["transform-async-to-module-method", {
```
Comment (rtrembecky) [inline .babelrc:8]: seems you can remove this package now: `babel-plugin-transform-async-to-module-method`
run `yarn remove babel-plugin-transform-async-to-module-method`

**Review by rtrembecky — COMMENTED** [review]
we can do some cleanup now

File: src/slack/slack.js (hunk 1 above)
Comment (mekan-allaberdi) [reply src/slack/slack.js:836]: we need `makeApiCall` for `logError` function now.

File: .babelrc (hunk 2 above)
Comment (mekan-allaberdi) [reply .babelrc:8]: removed `babel-plugin-transform-async-to-module-method `and pushed updated `package.json` and `yarn.lock`

Comment (rtrembecky) [reply .babelrc:8]: doesn't seem you pushed yet

**Review by rtrembecky — APPROVED** [review]
👍

**Pull request merged**